projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -297,20 +139,6 @@ ProductCard.propTypes = {
 	isManageDisabled: PropTypes.bool,
 	onActivate: PropTypes.func,
 	slug: PropTypes.string.isRequired,
-	showMenu: PropTypes.bool,
-	showActivateOption: PropTypes.bool,
-	showDeactivateOption: PropTypes.bool,
-	showInstallOption: PropTypes.bool,
-	menuItems: PropTypes.arrayOf(
-		PropTypes.shape( {
-			label: PropTypes.string,
-			icon: PropTypes.node,
-			onClick: PropTypes.func,
-		} )
-	),
-	onInstallStandalone: PropTypes.func,
-	onActivateStandalone: PropTypes.func,
-	onDeactivateStandalone: PropTypes.func,
```

Comment:
I notice by removing the menu that we are removing the action to install the standalone plugins from My Jetpack.

Just confirming that this is intended @eeeeeevon2023? Do we intend to provide another way for users to install the standalone plugins. If you don't have the main Jetpack plugin, there is an "install" option on each card, but if you do have the main Jetpack plugin, there's not a clear install path.

Since the standalone plugins offer the most updated UI and the latest features, I think we should still have a clear way to install them for each product from My Jetpack.

Reply:
>  I notice by removing the menu that we are removing the action to install the standalone plugins from My Jetpack.
Good catch. I noticed only today that the Figma file for the cards is out of date. The menu should not be removed. The rest of the tasks are correct. Let me rework this a bit tomorrow. Sorry-the covid brain fog has been really getting me lately. Give me tomorrow to sort this out!